build.gradle

```diff
@@ -74,12 +74,50 @@ jar {
     from sourceSets.api.output.resourcesDir
 }
 
-dependencies {
-    //to change the versions see the gradle.properties file
-    minecraft "com.mojang:minecraft:${project.minecraft_version}"
-    mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
-    modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
+repositories {
```

Comment:
This entire `repositories` block is not strictly required either but prevents downloading artifacts from the wrong place.